src/EditorFeatures/CSharpTest/RemoveUnusedParametersAndValues/RemoveUnusedParametersTests.cs

```diff
@@ -1290,6 +1290,35 @@ public async Task RegressionTest_ShouldReportUnusedParameter()
 
 public interface IFoo { event Action Fooed; }
 
+public sealed class C : IDisposable
+{
+    private readonly Task<IFoo> foo;
```

Comment:
can't use 'foo'.

Reply:
Ah, I copied an existing test. Let me change both the tests.